google/cloud/aiplatform/training_jobs.py

```diff
@@ -1015,6 +1097,62 @@ def __init__(
             model_serving_container_health_route
         )
 
+        self._model_serving_container_command = model_serving_container_command
+        self._model_serving_container_args = model_serving_container_args
+        self._model_serving_container_environment_variables = (
+            model_serving_container_environment_variables
+        )
+        self._model_serving_container_ports = model_serving_container_ports
+
+        self._model_description = model_description
+
+        model_predict_schemata = None
+        if any(
+            [
+                model_instance_schema_uri,
+                model_parameters_schema_uri,
+                model_prediction_schema_uri,
+            ]
+        ):
+            model_predict_schemata = gca_model.PredictSchemata(
+                instance_schema_uri=model_instance_schema_uri,
+                parameters_schema_uri=model_parameters_schema_uri,
+                prediction_schema_uri=model_prediction_schema_uri,
+            )
+
+        # Create the container spec
+        env = None
+        ports = None
+
+        if model_serving_container_environment_variables:
+            env = [
+                env_var.EnvVar(name=str(key), value=str(value))
+                for key, value in model_serving_container_environment_variables.items()
+            ]
+
+        if model_serving_container_ports:
+            ports = [
+                gca_model.Port(container_port=port)
+                for port in model_serving_container_ports
+            ]
+
+        container_spec = gca_model.ModelContainerSpec(
+            image_uri=model_serving_container_image_uri,
+            command=model_serving_container_command,
+            args=model_serving_container_args,
+            env=env,
+            ports=ports,
+            predict_route=model_serving_container_predict_route,
+            health_route=model_serving_container_health_route,
+        )
+
+        # create model payload
+        self._managed_model = gca_model.Model(
```

Comment:
Since this managed_model is created, I don't think we need to store the fields stored in the managed_model as attributes on this object.

ie:
self._model_serving_container_args 
self._model_serving_container_environment_variables
self._model_serving_container_ports
self._model_description
etc...
can be removed and any downstream instance method that may require these fields can access them through self._managed_model.

Reply:
Done